### PR TITLE
Add durham tools

### PR DESCRIPTION
**Pull request by annahaensch**
# What is this about? 

This PR adds the tools for pulling and pre-processing the data from the Durham, NC Police Department.  It's based on the tooling originally written in R by the SToPA-Durham lab.  The tools themselves live in 
```
SToPA/src/durham/
```
a merged version of the pdf data for 2019 has been added to google drive [here](https://drive.google.com/drive/folders/1t1FIzYdgJwvg_V14ZQS5hO50j7F_nplI?usp=share_link).  This are just to help with some initial exploratory data analysis.  I've also included an explanatory notebook for going through this pipeline yourself, which you can find in
```
SToPA/notebooks/Getting_Started_With_The_Durham_Data.ipynb.
```

### CODE REVIEW

**Review by samuelhansen — APPROVED** [review]

**Comment by samuelhansen** [comment]
Ooops, did not mean to approve the whole PR. I do have questions about the merge function as the data I was dealing with was a single report per PDF and that is how I wrote the extraction script. I am going to spend some more time going through those functions

**Comment by samuelhansen** [comment]
Do not merge this, there is a major issue

**Comment by annahaensch** [comment]
@samuelhansen just checking in, is this still in progress?  Or are you ready for a review?  I'm testing things out on my local install right now, and so far everything is running smoothly.

**Comment by samuelhansen** [comment]
I am not in process anymore, please feel free to review

**Review by annahaensch — COMMENTED** [review]
Overall this is really coming together.  I'm still working on reviewing, but I'll leave these initial comments for now.

File: src/durham/get_and_parse_reports.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+            success=True
+            break
+        else:
+            print(start.strftime("%m/%d/%Y"), 'scrape failed')
+            cnt += 1
+    #If the scrape for the date fails 10 times add it to a list
+    if not success:
+        failList.append(start.strftime("%m/%d/%Y"))
+
+print(failList)
```
[the remark this answers is not in the document]
Comment (annahaensch) [reply src/durham/get_and_parse_reports.py:308]: I resolved this, a script has been added at `scripts/durham/get_pdf_logs.py`

**Review by annahaensch — COMMENTED** [review]
This looks good to me for now.  I'm going to merge it, but @samuelhansen if you want to revisit any of this code feel free to open a new PR and we can work on it some more.

**Pull request merged**